gatsby-node.js

```diff
@@ -288,11 +298,7 @@ async function runIndexQueries(
  * @return {Promise}
  */
 async function moveIndex(client, sourceIndex, targetIndex) {
-  const { taskID } = await client.moveIndex(
-    sourceIndex.indexName,
-    targetIndex.indexName
-  );
-  return targetIndex.waitTask(taskID);
+  await client.moveIndex(sourceIndex.indexName, targetIndex.indexName);
```

Comment:
```suggestion
  return client.moveIndex(sourceIndex.indexName, targetIndex.indexName).wait();
```

and the function no longer needs to be async